M2/configure.ac

```diff
@@ -741,6 +742,28 @@ AC_COMPILE_IFELSE([AC_LANG_SOURCE([
 	# 1 "foo" 1 3 4
 	template <class T> struct A { };])],AC_MSG_RESULT(no);ISYSTEM=yes,AC_MSG_RESULT(yes))
 
+dnl Check for openmp and fail if it cannot be found.
+dnl This is required for building csdp.
+AC_MSG_CHECKING(whether OpenMP is supported by the compiler)
+if test ! "$OPENMP_LDFLAGS"
```

Comment:
Don't make the test conditional.  You should always test.

Reply:
I agree.